tensorrt_llm/_torch/visual_gen/models/wan/fp4_conv_autotuner.py

```
@@ -0,0 +1,233 @@
+        if do_preparation:
+            # CuTe compilation can be orders of magnitude slower than a launch;
+            # compile every candidate before the autotuner starts timing.
+            for tactic_id, candidate in enumerate(FP4_CONV_TACTICS):
+                try:
+                    self.compile_tactic(candidate)
+                except Exception as error:
+                    # CuTe compilation may touch CUDA before rejecting a
+                    # configuration; clear any pending error before continuing.
+                    try:
+                        torch.cuda.synchronize()
+                    except Exception as sync_error:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)